README.md

```diff
@@ -57,23 +57,117 @@ Right now the language server supports the following actions:
 
 - $/progress
 - textDocument/publishDiagnostics
+  - params: `types.PublishDiagnosticsParams`
+  - example: Snyk Open Source
+  ```json5
+  {
+    "uri": "file:///path/to/file",
+    "diagnostics": [
+      {
+        "range": {
+          "start": { "line": 1, "character": 0 },
+          "end": { "line": 2, "character": 0 },
+        },
+        "severity": 1,
+        "code": "S100",
+        "source": "Snyk",
+        "message": "Message",
+        "tags": ["security"],
+        "data": {
+          "scanIssue": {
+            "id": "123",
+            "issueType": "vulnerability",
+            "packageName": "packageName",
+            "packageVersion": "packageVersion",
+            "issue": "issue",
+            "additionalData": {
+              "ruleId": "ruleId",
+              "identifiers": {
+                "cwe": ["cwe"],
+                "cve": ["cve"]
+              },
+              "description": "description",
+              "language": "language",
+              "packageManager": "packageManager",
+              "packageName": "packageName"
+            }
+          }
+        }
+      }
+    ]
+  }
+  ```
+  - example: Snyk Code
+  ```json5
+  {
+    "uri": "file:///path/to/file",
+    "diagnostics": [
+      {
+        "range": {
+          "start": { "line": 1, "character": 0 },
+          "end": { "line": 2, "character": 0 },
+        },
+        "severity": 1,
+        "code": "S100",
+        "source": "Snyk",
+        "message": "Message",
+        "tags": ["security"],
+        "data": {
+          "scanIssue": {
```

Comment:
Is this correct? I thought it is the struct directly, without json `key`.

Reply:
You're right. Correction in https://github.com/snyk/snyk-ls/pull/699